indra/llwindow/llkeyboardmacosx.cpp

```diff
@@ -304,14 +271,4 @@ void LLKeyboardMacOSX::scanKeyboard()
 	}
 }
 
-BOOL LLKeyboardMacOSX::translateNumpadKey( const U16 os_key, KEY *translated_key )
-{
-	return translateKey(os_key, translated_key);
-}
-
-U16	LLKeyboardMacOSX::inverseTranslateNumpadKey(const KEY translated_key)
-{
-	return inverseTranslateKey(translated_key);
-}
-
```

Comment:
This was some very old cruft... `LLKeyboardMacOSX::translateNumpadKey()` was just a wrapper for `translateKeys()`.  Meanwhile, `LLKeyboardMacOSX::inverseTranslateNumpadKey()` was never called anywhere.  Similar cruftitude existed in other platform implementations.